### PR TITLE
Updated node migration scripts in CE 4.11 release notes

### PR DESCRIPTION
**Pull request by AnnaK-R3**
Preview: https://anna.preview.docs.r3.com/en/platform/corda/4.11/community/release-notes.html#database-schema-changes

### CODE REVIEW

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 1]
```diff
@@ -212,10 +212,10 @@ The following database changes have been applied:
   @Column(name = "states_to_record")
   val statesToRecord: StatesToRecord? = null
   ```
-  See the following node migration scripts:
-  * `node-core.changelog-v24.xml`: added transaction signatures.
-  * `node-core.changelog-v24.xml`: added finality flow recovery metadata.
-* The `vault_state` table now includes a `consuming_tx_id` column.
+  See node migration scripts:
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:215]: Hard to tell without a preview but is it correct to indent this?

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 2]
```diff
@@ -212,10 +212,10 @@ The following database changes have been applied:
...
+  * `node-core.changelog-v25.xml`: Adds Sender and Receiver recovery distribution record tables, plus the `PartyInfo` table.
+  * `node-core.changelog-v26.xml`: Adds AES encryption keys table.
+  * The `vault_state` table now includes a `consuming_tx_id` column.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:218]: Not sure that this should be indented? It's not a migration script?

File: content/en/platform/corda/4.11/community/release-notes.md [hunk 3]
```diff
@@ -105,6 +105,15 @@ This release includes the following fixes:
 
 The following database changes have been applied:
 
+* The `vault_state` table now includes a `consuming_tx_id` column. The new column was added in the following migration script: `vault-schema.changelog-v14.xml`.
+
+* Two Phase Finality introduces additional data fields within the main `DbTransaction` table:
```
Comment (colljos) [inline content/en/platform/corda/4.11/community/release-notes.md:110]:
```suggestion
* Two Phase Finality introduces an additional data field within the main `DbTransaction` table:
```

File: content/en/platform/corda/4.11/community/release-notes.md [hunk 4]
```diff
@@ -189,9 +198,9 @@ class EncryptionKeyRecord(
 ```
 
 See node migration scripts:
- * `node-core.changelog-v25.xml`: Adds Sender and Receiver recovery distribution record tables, plus the PartyInfo table.
- * `node-core.changelog-v26.xml`: Adds AES encryption keys table.
-* The `vault_state` table now includes a `consuming_tx_id` column.
+* `node-core.changelog-v23.xml`: Adds additional data fields within the main `DbTransaction` table.
```
Comment (colljos) [inline content/en/platform/corda/4.11/community/release-notes.md:201]:
```suggestion
* `node-core.changelog-v23.xml`: Adds an additional data field within the main `DbTransaction` table.
```

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 5]
```diff
@@ -187,35 +187,102 @@ This release includes the following fixes:
...
 
-  /** X500Name of flow initiator **/
-  @Column(name = "initiator")
-  val initiator: String? = null,
-
-  /** X500Name of flow participant parties **/
-  @Column(name = "participants")
-  @Convert(converter = StringListConverter::class)
-  val participants: List<String>? = null,
+* Two Phase Finality introduces additional data fields within the main `DbTransaction` table:
```
Comment (colljos) [inline content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:192]:
```suggestion
* Two Phase Finality introduces an additional data field within the main `DbTransaction` table:
```

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 6]
```diff
@@ -187,35 +187,102 @@ This release includes the following fixes:
...
+        @Column(name = "key_id", nullable = false)
+        val keyId: UUID,
+
+        @Column(name = "key_material", nullable = false)
+        val keyMaterial: ByteArray
+)
+```
+
+See node migration scripts:
+* `node-core.changelog-v23.xml`: Adds additional data fields within the main `DbTransaction` table.
```
Comment (colljos) [inline content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:283]:
```suggestion
* `node-core.changelog-v23.xml`: Adds an additional data field within the main `DbTransaction` table.
```

**Review by colljos — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md]: Generated a preview.

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md]: Generated a preview. Updated section after checking with Paul and Jose.

**Review by colljos — APPROVED** [review]

File: content/en/platform/corda/4.11/community/release-notes.md [hunk 7]
```diff
@@ -105,6 +105,15 @@ This release includes the following fixes:
 
 The following database changes have been applied:
 
+* The `vault_state` table now includes a `consuming_tx_id` column. The new column was added in the following migration script: `vault-schema.changelog-v14.xml`.
+
+* Two Phase Finality introduces an additional data field within the main `DbTransaction` table:
+
+```kotlin
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/community/release-notes.md:112]: This code block should be indented in line with the bullet that it relates to.

File: content/en/platform/corda/4.11/community/release-notes.md [hunk 8]
```diff
@@ -105,6 +105,15 @@ This release includes the following fixes:
...
+@Column(name = "signatures")
+val signatures: ByteArray?
+```
+
 * Two Phase Finality introduces two new database tables for storage of recovery metadata distribution records:
 
 ```bash
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/community/release-notes.md:119]: This code block should be indented in line with the bullet that it relates to.

**Review by nadinequinr3 — COMMENTED** [review]
I'm not sure about the pieces above "See node migration scripts". For example, should "There are two further tables..." be another bullet? or maybe it just all needs to be indented to be show as part of the "Two Phase Finality introduces.." bullet?

File: content/en/platform/corda/4.11/community/release-notes.md (hunk 7 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.11/community/release-notes.md:112]: Indented the code.

File: content/en/platform/corda/4.11/community/release-notes.md (hunk 8 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.11/community/release-notes.md:119]: Indented the code.

**Comment by AnnaK-R3** [comment]
> I'm not sure about the pieces above "See node migration scripts". For example, should "There are two further tables..." be another bullet? or maybe it just all needs to be indented to be show as part of the "Two Phase Finality introduces.." bullet?

Yes, all of this is part of the Two Phase Finality bullet. Indented the content accordingly so it's more clear.

**Review by nadinequinr3 — APPROVED** [review]

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**